contracts/modules/ProtocolSettings.sol

```diff
@@ -234,6 +236,19 @@ contract ProtocolSettings is State, ProtocolTokenUser, ProtocolSettingsEvents, M
 		emit SetBorrowingFeePercent(msg.sender, oldValue, newValue);
 	}
 
+	/**
+	 * @notice Set the value of swapExtrernalFeePercent storage variable
+	 *
+	 * @param newValue the new value for swapExternalFeePercent
+	 */
+	function setSwapExternalFeePercent(uint256 newValue) external onlyOwner whenNotPaused {
```

Comment:
consider giving the `newValue` parameter a more meaningful name.
// there are multiple setters with  `newValue` parameter